index.html

```diff
@@ -12,5 +12,7 @@
     <button id="zerar" type="button">zerar</button>
     <button id="decrementar" type="button">decrementar</button>
     <script src="contador.js"></script>
+
+    <h1>Alteração no codigo pelo dev</h1>
```

Comment:
```suggestion
    <h2>Alteração no codigo pelo dev</h2>
```
Não é um H1, pois é muito grande e pode causar estranheza no final do site